docs/user-guide.md

```diff
@@ -336,6 +336,46 @@ In the file menu, you can choose to unshare it:
 1. In the Manage attachments dialog, point to a file and select it. You can select several files.
 2. Click the **Delete** icon below the files tree.
 
+### Publish
+
+You can publish conversations to make them available for the target audience. You can also unpublish them to withdraw from public use.
+
+**Note**: to publish a shared conversation, [duplicate](#duplicate) it and then publish. 
+
+**To publish a conversation**: 
+
+1. Click **Publish** in the conversation menu. 
+
+![](./img/publish-conversation.png)
+
+2. In the pop-up window:
+   * Enter **publication request name** in the upper area.
+   * In **Publish to**, you can select the target audience. You can also create a dedicated folder to publish your conversation into.
```

Comment:
'Publish to' indicates only where we put the publication, i.e. the destination folder. We define the target audience in 'allow access...'

Reply:
fixed